src/clj/rems/application/cleaner.clj

```diff
@@ -0,0 +1,46 @@
+(ns rems.application.cleaner
```

Comment:
name pending. suggestions are welcomed

Reply:
Terminator or Eraser?

Reply:
i like where you are heading with this. eraser gets my vote